admin_manual/configuration_server/harden_server.rst

```diff
@@ -43,6 +47,17 @@ It is highly recommend to enable hardening modules such as SELinux where
 possible. See :doc:`../installation/selinux_configuration` to learn more about 
 SELinux.
 
+Use newest PHP version
+**********************
+Some security features and hardenings in PHP are only available in the latest
+PHP release and thus it is recommended to run the latest PHP version. You can
+find the latest version on http://php.net/.
+
+While distributions are trying to backport security patches there have been
+quite some cases where some has been missed or only backported after a long
+period of time, sometimes up to a few years. Also all security hardenings
+are usually not backported.
+
```

Comment:
Changed. Replaced  "many" with "some" as I don't want to deal with vendors complaining to us :see_no_evil: